src/pages/invest_KYC6.js

```diff
@@ -0,0 +1,287 @@
+import {
```

Comment:
This file needs to be renamed to Invest_KYC6.js.  The lowercase "i" is causing issues.

Reply:
i already changed small i to capital I, but it's not showing in github, any solution please

Reply:
> i already changed small i to capital I, but it's not showing in github, any solution please
Yes, try this please:
git mv src/pages/invest_KYC6.js src/pages/Invest_KYC6.js